src/GlobalStyles.js

```diff
@@ -4,6 +4,8 @@ import { createGlobalStyle } from 'styled-components'
 export default createGlobalStyle`
 :root {
   --border-radius-global: 50px;
+  --border-radius-form: 20px;
+  --border-radius-form: 10px;
```

Comment:
The second value `--border-radius-form` is overwriting the first one. You only need this once!

Reply:
Oh yes :)